homeassistant/components/intellifire/sensor.py

```diff
@@ -0,0 +1,132 @@
+"""Platform for sensor integration."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+
+from intellifire4py import IntellifirePollData
+
```

Comment:
Double import, guess that happened during you making changes and I doing review.
```suggestion
```

Reply:
Caught it on my end!